lib/puppet/indirector/trusted_external/exec.rb

```diff
@@ -0,0 +1,19 @@
+require 'puppet/trusted_external'
+require 'puppet/indirector/exec'
+require 'json'
+
+class Puppet::TrustedExternal::Exec < Puppet::Indirector::Exec
+  desc "Call an external program to get extended trusted facts information."
+  include Puppet::Util
+
+  def command
+    command = Puppet[:trusted_external_command]
+    raise ArgumentError, _("You must set the 'trusted_external_command' parameter to use the extended trusted facts terminus") unless command != "none"
+    command.split
```

Comment:
If this is supposed to run on Windows, `command.split` is probably a very naive way to split a string into an array of strings to execute. For example; How would you pass `C:\Program Files\ServiceNow\fetchnode.exe` ?